docs/howto/use-alternative-secrets-backend.rst

```diff
@@ -52,7 +52,105 @@ Set ``backend`` to the fully qualified class name of the backend you want to ena
 You can provide ``backend_kwargs`` with json and it will be passed as kwargs to the ``__init__`` method of
 your secrets backend.
 
-See :ref:`AWS SSM Parameter Store <ssm_parameter_store_secrets>` for an example configuration.
+.. _local_filesystem_secrets:
+
+Local Filesystem Secrets Backend
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This backend is especially useful in the following use cases:
+
+* **development**: It ensures data synchronization between all terminal windows (same as databases),
+  and at the same time the values are retained after database restart (same as environment variable)
+* **Kubernetes**: It allows you to store secrets in `Kubernetes Secrets <https://kubernetes.io/docs/concepts/configuration/secret/>`__
+  or you can synchronize values using the sidecar container and
+  `a shared volume <https://kubernetes.io/docs/tasks/access-application-cluster/communicate-containers-same-pod-shared-volume/>`__
+* **JSON**: If you're tired of defining all connections using a URI - creating JSON object is easier than using URI.
```

Comment:
```suggestion
```

I don't think this is a real-usecase

Reply:
Development and K8s Secrets are fine (as K8s Secrets would be encrypted) but other than that storing Secrets in File (in plain-text) shouldn't be encouraged.